frontend/app/views/fragments/event/item.scala.html

```diff
@@ -7,7 +7,7 @@
     <div class="event-item__container">
         <div class="event-item__media">
             <a href="/event/@event.id" itemprop="image" content="@event.socialImgUrl">
-                <div class="delayed-image-load" data-src="@event.imgUrl" data-alt="@event.name.text" data-class="responsive-img"></div>
+                @fragments.event.image(event)
```

Comment:
Nice, I've got some work about to go up that splits the this item fragment up into two versions so it's good to have this as a standalone fragment.